libs/island-ui/core/src/lib/Icon/Icon.tsx

```diff
@@ -274,6 +273,9 @@ const SvgPathContainer = ({
   )
 }
 
+/**
+ * @deprecated Use something else?
```

Comment:
There is a newer Icon component, IconRC. We should point people there.

```suggestion
 * @deprecated Use IconRC instead
```